### ISSUE TITLE
How to log metric value in checkpoint file name with default save dir?

### ISSUE THREAD

**Issue by xiadingZ** [PyTorchLightning/pytorch-lightning#4168]
Now checkpoint default save to dir named with job id, such as `version_1986332/checkpoints/epoch=415.ckpt`, I want to  save it as `version_1986332/checkpoints/epoch=415-loss=0.1.ckpt`. I know I can set `filepath='my/path/sample-mnist-{epoch:02d}-{val_loss:.2f}'`, but it won't automatically save to `version_{job id}` dir. How can I only change checkpoints's filename?

**Comment by rohitgr7** [comment]
I believe https://github.com/PyTorchLightning/pytorch-lightning/issues/3583 will solve this issue.
